package.json

```diff
@@ -39,9 +39,9 @@
     "prettier:fix": "yarn prettier --write",
     "prettier": "prettier './src/**/*.{js,jsx,ts,tsx}'",
     "release": "electron-builder --mac --linux --windows -p always",
-    "start-mainnet": "REACT_APP_NETWORK=mainnet yarn start",
-    "start-polygon": "REACT_APP_LATEST_SAFE_VERSION='1.3.0' REACT_APP_NETWORK=polygon yarn start",
-    "start-bsc": "REACT_APP_LATEST_SAFE_VERSION='1.3.0' REACT_APP_NETWORK='bsc' yarn start",
+    "start-mainnet": "cross-env REACT_APP_NETWORK=mainnet yarn start",
+    "start-polygon": "cross-env REACT_APP_LATEST_SAFE_VERSION='1.3.0' REACT_APP_NETWORK=polygon yarn start",
+    "start-bsc": "cross-env REACT_APP_LATEST_SAFE_VERSION='1.3.0' REACT_APP_NETWORK='bsc' yarn start",
```

Comment:
`cross-env` is probably an npm package und thus needs to be installed?

Reply:
I installed WSL a couple days ago, so I will revert these changes because they only applied to me

Reply:
Coming back to this briefly, it is already installed for a few other commands.